src/TinyGPS++.cpp

```diff
@@ -355,14 +355,14 @@ void TinyGPSLocation::setLongitude(const char *term)
 double TinyGPSLocation::lat()
 {
    updated = false;
-   double ret = rawLatData.deg + rawLatData.billionths / 1000000000.0;
+   double ret = (double)rawLatData.deg + ((double)rawLatData.billionths / (double)1000000000.0);
```

Comment:
Does this really make a difference?
The `1000000000.0` (thus without a trailing `f`) already should imply a double value.

Reply:
Agree.  Thanks for the comment, @TD-er.  Disinclined to make alterations until someone can prove they are necessary.

Reply:
Maybe if there is some compiler or compiler flag to explicitly disable double support?
Or switching the order so the division is done first and then the add?

The reason I did question this was not to tell you not to merge it, but more like a genuine surprise.
It is clear this is the expected and intended code, so I am quite surprised if this would make any difference.